javascript/node/selenium-webdriver/lib/webdriver.js

```diff
@@ -1131,6 +1131,16 @@ class Options {
       expiry = Math.floor(date.getTime() / 1000);
     }
 
+    // if not declared, attribute is set to 'undefined'
+    // (or)
+    // if declared (ex: sameSite:"") and string length is zero, attribute is 'None' by default
+    if(typeof sameSite === "undefined" || sameSite.length===0) {
```

Comment:
I would only check if the value is `===` `Lax` or `Strict`, there is no other value allowed. So if someone sends an empty string, an error should be thrown.

Reply:
Hi @christian-bromann,

I've updated code as suggested. Can you please review again

Thanks,
Harsha